### ISSUE TITLE
Show skill specialisations (“Fertigkeitsspezialisierung”) and scope of application (“Anwendungsgebiete”) in the skill list ("Talente")

### ISSUE THREAD

**Issue by mwberlindeu** [elyukai/optolith-client#1183]
Showing the skill specialisations (“Fertigkeitsspezialisierung”) and and the scope of application (“Anwendungsgebiete”) in the skill list ("Talente") would improve the density of information on a character's skill. It is also easier to read as the information is visible in one place instead of three.

Currently skill specialisations (“Fertigkeitsspezialisierung”) are shown in the special abilities (“Allgemeine Sonderfertigkeiten”).
The scope of application (“Anwendungsgebiete”) of a skill is not available and has to be looked up in the rules.
Special abilities (“Allgemeine Sonderfertigkeiten”) that extend the scope of application (“Anwendungsgebiete”) of a skill is only visible in the special abilities (“Allgemeine Sonderfertigkeiten”) section.

All this information could be condensed in an updated skill list.

see also:
https://gist.github.com/elyukai/9152a4985c2c1384840eeadaf451d848#gistcomment-3700465
https://gist.github.com/elyukai/9152a4985c2c1384840eeadaf451d848#gistcomment-3700474

**Comment by elyukai** [comment]
Related: #696

Default applications are too much to include them on the character sheet (especially for skills like geography where you have this huge list of regions), but New Applications (Neue Anwendungsgebiete) and Uses (Einsatzmöglichkeiten), as well as the effects of special abilities such as Skill Specialization would be helpful.

**Comment by mwberlindeu** [comment]
Well, for majority of skills you just have 3-5 if my memory is not failing. You may omit them if you have a really long list like "Geographie", "Rechtskunde", "Götter/Kulte", etc.

**Comment by elyukai** [comment]
Depends on how much pages you want to use to print all the different skills. They would usually cover 1–3 lines (excluding long lists) which would result in about 3 pages for skills instead of 1.

**Comment by mwberlindeu** [comment]
Yes, that is essentially a design decision. I would say you can do with one extra line per skill and a small Font) but it at least doubles the amount of space you need.
